### ISSUE TITLE
Missing Schema Type

### ISSUE THREAD

**Issue by alvis** [ardatan/graphql-import#190]
Seems like `importSchema` omits schema type at the moment. For example, I'd expect the parsed SDL in the following simple code contains the schema type and be idential to the source. However, it doesn't.

```typescript
import { importSchema } from 'graphql-import';

const typeDefs = `
schema {
  query: Query
}

type Query {
  me: String
}
`;

it('should be the same', () => {
  expect(importSchema(typeDefs)).toEqual(typeDefs);
});
```

When you run jest, here is the result:
```
 Expected value to equal:
      "
    schema {
        query: Query
    }

    type Query {
        me: String
    }
    "
    Received:
      "type Query {
      me: String
    }
    "
```

This expected behaviour is hitting me badly as missing the schema type means `visitSchema` in `grahpql-tools` cannot be triggered. I've a hacky solution to my problem, but I think it is a bug which should be fixed.

**Comment by patsissons** [comment]
this is definitely a bug, the following assertion should not fail.

```ts
const sdl = 'schema { query: QueryRoot } type QueryRoot { foo: String }';
expect(print(parse(importSchema(sdl)))).toEqual(print(parse(sdl)));
```

**Comment by gerardolima** [comment]
Is this repository dead or this bug wasn't considered not woth solving?

**Comment by ardatan** [comment]
Hi @gerardolima  !
In 1.0.0 beta release, we introduced a lot of changes including schema definition support;
Could you install graphql-import@beta to try new changes? Don't forget to modify your code regarding to the migration notes in README.
https://github.com/ardatan/graphql-import#updating-from-07x

**Comment by ardatan** [comment]
Available in 1.0.0!
